documentation/src/main/resources/pages/ditto/installation-operating.md

```diff
@@ -16,6 +16,37 @@ This page shows the basics for operating Ditto.
 Ditto has many config parameters which can be set in the config files or via environment variables.
 This section will cover some of Ditto's config parameters.
 
+### Migrating to Ditto 3.4.x
```

Comment:
@alstanchev this section should be moved to a new `release_notes_340.md`.

We always documented the migration notes in the release notes.
At the current location, probably they are not found.